### PR TITLE
Druid Quickstart

### PR DESCRIPTION
**Pull request by RamanaReddy8801**
# Summary
Ticket: https://new-relic.atlassian.net/browse/NR-234861
Description: Created new Apache Druid quickstart

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [X] Did you check you NRQL syntax? - Does it work?
- [X] Did you include a Data source and Documentation reference?
- [X] Are all documentation links publicly accessible?
- [X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [X] Did you check your descriptive content for spelling and grammar errors?
- [X] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [X] Does the PR contain a screenshot for each of your dashboards?
- [X] Do your screenshots show data?
- [X] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [X] Did you check that your alerts actually work?

### CODE REVIEW

File: alert-policies/apache-druid/jvmMemoryUsage.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 90
```
Comment (pkudikyala) [inline alert-policies/apache-druid/jvmMemoryUsage.yml:18]:
```suggestion
    threshold: 95
```

File: alert-policies/apache-druid/jvmMemoryUsage.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    # Time in seconds; 120 - 3600
+    thresholdDuration: 300
+    # How many data points must be in violation for the duration
+    thresholdOccurrences: ALL
+  - priority: WARNING
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 80
```
Comment (pkudikyala) [inline alert-policies/apache-druid/jvmMemoryUsage.yml:27]:
```suggestion
    threshold: 90
```

**Review by pkudikyala — CHANGES_REQUESTED** [review]
Hi @RamanaReddy8801 ,
Could you please make the review changes.

**Review by sjyothi54 — APPROVED** [review]
LGTM

**Review by pkudikyala — APPROVED** [review]
LGTM!

**Pull request merged**